greenheart/tools/eco/electrolysis.py

```diff
@@ -76,7 +101,7 @@ def run_electrolyzer_physics(
     # store results for return
     electrolyzer_physics_results = {
         "H2_Results": H2_Results,
-        "electrical_generation_timeseries": energy_to_electrolyzer_kw,
+        "electrical_generation_timeseries": energy_input_to_electrolyzer,
```

Comment:
Please keep the output name the same as it was, including units

Reply:
these are different for grid cases because energy_to_electrolyzer_kw is not input for grid-connected cases and is output from the electrolyzer model. For off-grid, energy_to_electrolyzer_kw is the same as energy_input_to_electrolyzer. I can rename to make it more clear